cmd/gossamer/toml_config.go

```diff
@@ -46,7 +57,7 @@ func loadConfig(cfg *ctoml.Config, fp string) error {
 		},
 	}
 
-	if err = tomlSettings.NewDecoder(file).Decode(cfg); err != nil {
+	if err := tomlSettings.NewDecoder(file).Decode(cfg); err != nil {
```

Comment:
supermeganit, you could change the function signature (already changed for other reasons) to 

```go
func loadConfig(cfg *ctoml.Config, file fs.File) (err error) {
```

that way you can keep this line untouched. Verysupermeganit please forgive me 😄